rpm/python3-pytz.spec

```diff
@@ -0,0 +1,34 @@
+Summary:       Pytz
+Name:          pytz
+Version:       2022.1
+Release:       1
+License:       MIT
+Source0:       %{name}-%{version}.tar.gz
+URL:           https://pypi.org/project/pytz/
+Patch0:        0001-Use-system-zoneinfo.patch
+
+BuildRequires: python3-devel
+BuildRequires: python3-setuptools
+BuildRequires: rsync
+
+%description
+pytz brings the Olson tz database into Python
+
+%prep
+%autosetup -p1 -n %{name}-%{version}/%{name}
+
+%build
+make build
+cd src
+%{__python3} setup.py build
```

Comment:
Did the python rpm macro not work here?

Reply:
I guess you mean %python3, as we don't have %python (well, strictly speaking we do, but that doesn't work). Yes, %python3 works, I'll use that.

Reply:
I meant `%py3_build` and so on.

https://github.com/sailfishos-mirror/python-rpm-macros/blob/15e32a300599a69f29d060abe989c4b39f954060/macros.python3#L13

Reply:
@Thaodan I must say you often give quite some space for guessing... :wink: In any case good suggestions of course!

Reply:
Right, of course. I was thrown off track earlier because setup.py is not located in root but in src. But %py3_build works well after just changing directories.